### ISSUE TITLE
nav-bar: can't move focus out of md-nav-bar with shift+tab

### ISSUE THREAD

**Issue by frkfir** [angular/material#10419]
**Actual Behavior**:
 -  `What is the issue? * You can't focus out of md-nav-bar with shift+tab on keyboard.
 -  `What is the expected behavior? You should be able to focus out to other elements on the page from md-nav-bar with shift+tab.

**CodePen** (or steps to reproduce the issue): *
 -  `CodePen Demo which shows your issue: md-nav-bar demo on "Angular Material" site.
 -  `Details:`

**Angular Versions**: *
 -  `Angular Version: 1.5.5
 -  `Angular Material Version: 1.1.3

**Additional Information**:
 -  `Browser Type: * Chrome , Firefox and IE
 -  `Browser Version: * Chrome 56.0.2924.87 (64-bit)
 -  `OS: * Windows 7
 -  `Stack Traces: n/a

----
Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
  Add more lines if needed.

**Comment by frkfir** [comment]
@ErinCoughlan

**Comment by AShubin** [comment]
hey guys, any updates? have same issue

**Comment by frkfir** [comment]
Unfortunately not yet

**Comment by EladBezalel** [comment]
Please provide a codepen so we can investigate further.

**Comment by Gutsal** [comment]
The issue is still reproducible. 

Here is an example:
See CodePen demo provided by https://material.angularjs.org/latest/demo/navBar

AngularJS Material Version: 1.1.7

**Comment by Splaktar** [comment]
Yes, this is clearly reproducible in the docs. There are some other components that have this issue as well. Making this higher priority since it effects a11y.

**Comment by Splaktar** [comment]
This is still a problem, even with the a11y updates to nav-bar in PR https://github.com/angular/material/pull/11486.

**Comment by Splaktar** [comment]
@tirana I've opened PR https://github.com/angular/material/pull/11494 to fix this. Please take a look.

**Comment by tirana** [comment]
amazing. thank you!
